Library/Formula/vis.rb

```diff
@@ -0,0 +1,25 @@
+class Vis < Formula
+  desc "Modern, legacy free, simple yet efficient vim-like editor"
+  homepage "http://www.brain-dump.org/projects/vis"
+  url "http://www.brain-dump.org/projects/vis/vis-0.2.tar.gz"
+  sha256 "3e5b81d760849c56ee378421e9ba0f653c641bf78e7594f71d85357be99a752d"
+  head "https://github.com/martanne/vis.git"
+
+  keg_only "Shadows OS X' `vis` command"
+
+  depends_on "libtermkey"
+  depends_on "lua"
+  depends_on "lpeg" => :lua
```

Comment:
This needs to be vendored in the formula; we don't really support language dependencies like this in core any more due to the crappy user experience.